hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/leaky/LeakyCryptoTestsSuite.java

```diff
@@ -1019,6 +1022,70 @@ private HapiSpec hollowAccountCompletionWithEthereumTransaction() {
                                 }));
     }
 
+    private HapiSpec contractDeployAfterEthereumTransferLazyCreate() {
```

Comment:
Is this test needed for ContractCall and ContractCreate also?

Also we should update the test plan to make sure this coverage will be added for the future CryptoCreate via alias flow later.

Reply:
The test [is for ContractCreate](https://github.com/hashgraph/hedera-services/pull/4973/files#diff-6ed9c277a2c3aaf6aa4e75795718d93a5ee482807675d88d13fdd3771a67059dR1070), do you mean if we need an additional test for ContractCall as well? Or you mean something different related to ContractCreate?
For ContracCall I think, yes, there is similar logic for it and it should be affected and tested as well.

Reply:
The test name notes `EthereumTransaction`.
I'm asking is we need the same test for a ContractCall and a ContractCreate in which a hollow account is created.
I guess the later both require a contract execution which would include transfer to a new address.

Reply:
You're right in that the transaction type between the 3 may not matter is they all simplify down to the same transfer path logic.
The main thing would be making sure we have enough tests around whatever that list logic is to make sure population and clearing it done right